.dockerignore

```diff
@@ -1,7 +1,7 @@
-.git/*
-conf/*
-packager/*
-scripts/*
+.git
+conf
+packager
+scripts
```

Comment:
While testing using the `conf/*` form, the `conf` folder was still sent to the build context, making the symlink of `/data/gogs/conf` to `/app/gogs/conf/conf` as an empty `conf` folder was created.

This way the `conf` folder is not sent to the build context.

I admit that this is pretty weird, as `dockerignore` and `gitignore` does not really work exactly in the same way.

`gitignore` works great with the `conf/*` form as an empty folder is not sent to remote, but this is not the case with the docker build context.